diff --git a/include/reactphysics3d/configuration.h b/include/reactphysics3d/configuration.h
index f9443c6a..f3b20b27 100644
--- a/include/reactphysics3d/configuration.h
+++ b/include/reactphysics3d/configuration.h
@@ -33,6 +33,7 @@
 #include <utility>
 #include <sstream>
 #include <string>
+#include <vector>
 #include <cmath>
 #include <reactphysics3d/decimal.h>
 #include <reactphysics3d/containers/Pair.h>
